spec/presenters/tree_builder_archived_spec.rb

```diff
@@ -0,0 +1,103 @@
+describe TreeBuilderArchived do
+  context 'TreeBuilderArchived' do
+    before do
+      extend TreeBuilderArchived
```

Comment:
Curious why this is a module anyway..TreeBuilderImages, for example, is a class.